x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/permission/ResourcePrivileges.java

```diff
@@ -69,7 +69,7 @@ public static Builder builder(String resource) {
 
     public static final class Builder {
         private final String resource;
-        private Map<String, Boolean> privileges = new HashMap<>();
+        private Map<String, Boolean> privileges = new TreeMap<>();
```

Comment:
Yes, for consistent order. There were some tests in the `RBACEngine` that fail otherwise, because I changed them to randomize the privileges-to-check order (also add more privileges to check).